spring-webflux/src/main/java/org/springframework/web/reactive/socket/client/ReactorNettyWebSocketClient.java

```diff
@@ -102,6 +102,14 @@ public HttpClient getHttpClient() {
 				.next();
 	}
 
+	public int getMaxFramePayloadLength() {
+		return maxFramePayloadLength;
+	}
+
+	public void setMaxFramePayloadLength(int maxFramePayloadLength) {
+		this.maxFramePayloadLength = maxFramePayloadLength;
+	}
+
```

Comment:
In case you want to update your PR, these would need to be ordered earlier where getters an setters are, i.e. before implementation methods. Probably right ahead after getHttpClient(). Setters come first and some Javadoc on such a property is required.

Reply:
I'll go through the contribution guidelines and update this PR. Thanks for pointing that out.

Reply:
@rstoyanchev PR comments have been addressed.